packages/react-components/theme-designer/src/ThemeDesigner.tsx

```diff
@@ -1,11 +1,21 @@
 import * as React from 'react';
 import { makeStyles } from '@griffel/react';
 import type { ThemeDesignerProps } from './ThemeDesigner.types';
-import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
-import { createLightTheme, createDarkTheme } from '@fluentui/react-theme';
+import { FluentProvider } from '@fluentui/react-components';
+import {
+  createLightTheme,
+  createDarkTheme,
+  teamsLightTheme,
+  teamsDarkTheme,
+  webLightTheme,
+  webDarkTheme,
+  BrandVariants,
+  Theme,
+} from '@fluentui/react-theme';
 import { getBrandTokensFromPalette } from './utils/getBrandTokensFromPalette';
-import { brandWeb, brandTeams, brandOffice } from './utils/brandColors';
-import { useDebounce } from './utils/useDebounce';
+import { brandWeb, brandTeams } from './utils/brandColors';
```

Comment:
_nit:_ Ditto about alphabetical order

```suggestion
import { brandTeams, brandWeb } from './utils/brandColors';
```